app/helpers/repositories_helper.rb

```diff
@@ -155,15 +155,16 @@ def to_utf8_internal(str)
     end
     str = replace_invalid_utf8(str)
   end
+
   private :to_utf8_internal
 
   def replace_invalid_utf8(str)
     return str if str.nil?
     if str.respond_to?(:force_encoding)
       str.force_encoding('UTF-8')
-      if ! str.valid_encoding?
+      if !str.valid_encoding?
         str = str.encode("US-ASCII", invalid: :replace,
-              undef: :replace, replace: '?').encode("UTF-8")
+                         undef: :replace, replace: '?').encode("UTF-8")
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.